tests/iele/unit/bits.iele

```diff
@@ -0,0 +1,13 @@
+contract test {
+  define public @test() {
+    %ret = twos 0, 5
```

Comment:
Are you sure there are no regression test cases with nonzero width?

Reply:
I don't follow what you're asking here.

Reply:
Do the changes to the `sext` instruction change any results with a positive width? If so, that should also be covered for a regression test.

Reply:
okay, I will add a test for this.